gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -223,6 +223,8 @@ notifiers:
     sslTrustAll: false
     #sslKeyStore: /path/to/keystore
     #sslKeyStorePassword: changeme
+  licenseExpiration:
+    enabled: true
```

Comment:
**thought**: I'm not convince that this setting should be here. The `notifiers` section is here to declare how notifications can be sent to a user (email, UI, ...) but this settings is more about enabling or not the license notification... :thinking: Maybe we should introduce a "license" section (or "enterprise" section) :thinking: In any case, we have to be align with APIM as much as possible as this license notification will be present in both products.